tests/OpenTK.Tests/Box3Tests.fs

```diff
@@ -158,25 +158,42 @@ module Box3 =
             
     [<Properties(Arbitrary = [|typeof<OpenTKGen>|])>]
     module Inflate =
+
+        // See #1616 and PR 1662 -- it appears the test-runner generates F# 64-bit float inputs and
+        // C# 32-bit floats sometimes cause equivalency failures. Could possibly be fixed by updating
+        // Assertions.fs with Equal overloads that accept the precision argument, but per discussion on
+        // Discord the math tests may need re-thinking. In that case, how much variance is acceptable?
```

Comment:
I'll investigate this before merging this PR.

Reply:
F# was not doing anything with 64-bit floats. It was floating point imprecisions in the Inflate implementation that made the calculations not equal each other.